urdf/urdfdom_compatibility.h.in

```diff
@@ -41,7 +41,7 @@
 #define URDFDOM_HEADERS_MINOR_VERSION @URDFDOM_HEADERS_MINOR_VERSION@
 #define URDFDOM_HEADERS_REVISION_VERSION @URDFDOM_HEADERS_REVISION_VERSION@
 
-// for Wily: maintain compatibility between urdfdom 0.3 and 0.4 (definining SharedPtr types)
+// Define shared pointers for urdfdom versions less than 0.4
```

Comment:
Archaeological notes in https://github.com/ros/robot_model/pull/207/commits/0b0d2fcd720b3d3551d2881d32c2d968522cd4b4